src/contracts/whoopi/utils.ts

```diff
@@ -49,13 +45,11 @@ export const createWhoopiV0LendThunk = (
   }
   return await contract.lend(
     [nftAddress, tokenId, Array(tokenId.length).fill('0')],
-    upfrontRentFees.map((x, i) =>
-      toWhoopiScaledAmount(x, network, paymentTokens[i])
-    ) ?? [],
+    upfrontRentFees.map(({ value }) => value),
```

Comment:
The diff shows that upfront fee was scaled here

Reply:
@nazariyv My bad, my understanding was the expectation of this PR was for the caller to be the one who scales the value?